persistence-cassandra-4.0/src/main/java/io/stargate/db/cassandra/impl/interceptors/DefaultQueryInterceptor.java

```diff
@@ -253,10 +252,6 @@ private void applyState(
         // for DDL queries to reach agreement before returning.
         StargateSystemKeyspace.updatePeerInfo(
             endpoint, "schema_version", StargateSystemKeyspace.SCHEMA_VERSION);
-
-        // This fix schedules a schema pull for the non-member node and is required because
-        // `StorageService.onChange()` doesn't do this for non-member nodes.
-        MigrationManager.instance.scheduleSchemaPull(endpoint, epState);
```

Comment:
is it feasible to write a test (probably integration test) that checks that the schema from the non-member node is not pulled?

Reply:
I really don't think so, but I did prove it manually and by analyzing the C*/DSE code.